--- origsrc/monodevelop-vala-2.8.4.2/Makefile.am	2011-11-30 14:29:32.000000000 -0600
+++ src/monodevelop-vala-2.8.4.2/Makefile.am	2011-12-13 04:34:42.579447500 -0600
@@ -1,6 +1,6 @@
 ADDIN_BUILD = $(top_builddir)/build
 ASSEMBLY = $(ADDIN_BUILD)/MonoDevelop.ValaBinding.dll
-ASSEMBLY_CONFIG = MonoDevelop.ValaBinding.dll.config
+ASSEMBLY_CONFIG = $(srcdir)/MonoDevelop.ValaBinding.dll.config
 
 REFS =  \
 	$(GLIB_SHARP_LIBS) \
